worker/worker.go

```diff
@@ -122,10 +128,28 @@ func (w *Worker) startPolling(c <-chan os.Signal) {
 	for {
 		//TODO: Check github limits and run based on those
 		go w.getInitialIssues()
+		if w.checkRateLimitStatus() == true {
+			time.Sleep(time.Hour)
```

Comment:
Instead of time.Hour could you use the `resetAt` field that is returned from Github ( https://developer.github.com/v4/guides/resource-limitations/#returning-a-calls-rate-limit-status ) so we don't wait for more than we should?